src/snowcli/snow_connector.py

```diff
@@ -17,16 +18,22 @@
 TEMPLATES_PATH = Path(__file__).parent / "sql"
 
 
-def connect_to_snowflake(connection_name: Optional[str] = None, **overrides) -> SnowflakeConnection:  # type: ignore
-    connection_name = (
-        connection_name if connection_name is not None else get_default_connection()
-    )
-    try:
+def connect_to_snowflake(temporary_connection: bool = False, connection_name: Optional[str] = None, **overrides) -> SnowflakeConnection:  # type: ignore
+
+    if not temporary_connection:
+        connection_name = (
+            connection_name if connection_name is not None else get_default_connection()
+        )
         connection_parameters = cli_config.get_connection(connection_name)
-        if overrides:
-            connection_parameters.update(
-                {k: v for k, v in overrides.items() if v is not None}
-            )
+    else:
+        connection_parameters = {}
+
+    if overrides:
```

Comment:
Overrides are required for temporary connection. we should have at least 
- account
- users name and password or authenticator

Reply:
So, we should check if they`re present and raise specific error? Currently it`s handled by Python Connector, with an output, i felt, is clear enough (ex. "Invalid connection configuration. 251006: 251006: Password is empty")